recipes/taglib/all/conanfile.py

```diff
@@ -68,7 +68,10 @@ def _patch_sources(self):
             os.path.join(self.source_folder, "taglib", "CMakeLists.txt"),
             os.path.join(self.source_folder, "bindings", "c", "CMakeLists.txt"),
         ]:
-            replace_in_file(self, cmakelists, "INSTALL_NAME_DIR ${LIB_INSTALL_DIR}", "")
+            if self.version == 1.13:
+                replace_in_file(self, cmakelists, "INSTALL_NAME_DIR ${CMAKE_INSTALL_LIBDIR}", "")
+            if self.version == 1.12:
+                replace_in_file(self, cmakelists, "INSTALL_NAME_DIR ${LIB_INSTALL_DIR}", "")
```

Comment:
```suggestion
            if Version(self.version) >= "1.13":
                replace_in_file(self, cmakelists, "INSTALL_NAME_DIR ${CMAKE_INSTALL_LIBDIR}", "")
            else:
                replace_in_file(self, cmakelists, "INSTALL_NAME_DIR ${LIB_INSTALL_DIR}", "")
```

and add `from conan.tools.scm import Version`

Reply:
Thank you for your comment. I've updated my commit.